learning_table_tennis_from_scratch/models.py

```diff
@@ -1,36 +1,42 @@
 import pathlib
 
 from learning_table_tennis_from_scratch.hysr_one_ball_env import HysrOneBallEnv
-from learning_table_tennis_from_scratch.ppo_config import PPOConfig
-from learning_table_tennis_from_scratch.ppo_config import OpenAIPPOConfig
+from learning_table_tennis_from_scratch.rl_config import RLConfig
+from learning_table_tennis_from_scratch.rl_config import OpenAIRLConfig
 
 
 def run_stable_baselines(
     reward_config_file,
     hysr_one_ball_config_file,
-    ppo_config_file,
+    rl_config_file,
+    algorithm,
     log_episodes=False,
     seed=None,
 ):
     from stable_baselines3 import PPO
+    from stable_baselines3 import SAC
     from stable_baselines3.common import logger
     from stable_baselines3.common.env_util import make_vec_env
     from stable_baselines3.common.callbacks import CheckpointCallback
 
-    ppo_config = PPOConfig.from_json(ppo_config_file)
+    rl_config = RLConfig.from_json(rl_config_file, algorithm)
 
     tensorboard_logger = None
     checkpoint_callback = None
-    if ppo_config.log_path:
+    if rl_config.log_path:
         tensorboard_logger = logger.configure(
-            ppo_config.log_path, ["stdout", "csv", "tensorboard"]
+            rl_config.log_path, ["stdout", "csv", "tensorboard"]
         )
         tensorboard_logger.set_level(logger.INFO)
 
-        # Save a checkpoint every n_steps steps
+        # Save a checkpoint every n_steps steps, or every 10000 steps if n_steps does not exist (e.g. SAC)
+        try:
+            save_freq = getattr(rl_config, "n_steps", 10000)
+        except:
+            save_freq = 10000
```

Comment:
To my understanding, `getattr` shouldn't throw an error if a default value is specified, so the try/except may be unnecessary.